META-INF/com/google/android/update-binary

```diff
@@ -270,14 +390,37 @@ patch_libbluetooth() {
 
 ui_print " "
 print_full_bar
-print_justified "Multi-disabler v3.1 for Samsung devices"
+print_justified "Multi-disabler for Samsung devices"
```

Comment:
If this patch produces a new version, maybe it can be labeled as v3.2 ?

Reply:
Im no longer release it so no need for versioning